Documentation/Developer/InterlinkInventories.rst

```diff
@@ -33,6 +33,135 @@ This would output:
 *   :ref:`TYPO3 Explained 12 LTS <t3coreapi/v12:start>`
 *   :ref:`TYPO3 Explained 12.4 <t3coreapi/12.4:start>`
 
+..  _available-default-inventories:
+
+Currently available default inventories
```

Comment:
I would avoid "currently". Either it is "current" or "outdated". Additionally, when is "currently"?

Reply:
"Currently" is an adverb modifying the adjective "available," indicating that the inventories are available at the present time. Therefore, "currently available" functions as a compound adjective modifying "inventories." So "currently" is "at the time you are reading this".

Reply:
Ah, I thought you wanted to write "Current available default inventories". I'm fine with just writng "Available default inventories", that removes redundancy, ack!